### ISSUE TITLE
DAOs need rebuilding

### ISSUE THREAD

**Issue by twomice** [twomice/com.joineryhq.groupreg#9]
I'm seeing messages like this under civicrm 5.35.1:

```
User deprecated function: CRM_Groupreg_DAO_GroupregEvent needs to be regenerated. Missing getEntityTitle method. in CRM_Core_Error::deprecatedWarning() (line 1053 of /var/www/d7civi47latest/sites/all/modules/civicrm/CRM/Core/Error.php).
User deprecated function: CRM_Groupreg_DAO_GroupregEvent needs to be regenerated. Missing getEntityTitle method. in CRM_Core_Error::deprecatedWarning() (line 1053 of /var/www/d7civi47latest/sites/all/modules/civicrm/CRM/Core/Error.php).
User deprecated function: CRM_Groupreg_DAO_GroupregPriceField needs to be regenerated. Missing getEntityTitle method. in CRM_Core_Error::deprecatedWarning() (line 1053 of /var/www/d7civi47latest/sites/all/modules/civicrm/CRM/Core/Error.php).
User deprecated function: CRM_Groupreg_DAO_GroupregPriceField needs to be regenerated. Missing getEntityTitle method. in CRM_Core_Error::deprecatedWarning() (line 1053 of /var/www/d7civi47latest/sites/all/modules/civicrm/CRM/Core/Error.php).
User deprecated function: CRM_Groupreg_DAO_GroupregEvent needs to be regenerated. Missing getEntityTitle method. in CRM_Core_Error::deprecatedWarning() (line 1053 of /var/www/d7civi47latest/sites/all/modules/civicrm/CRM/Core/Error.php).
User deprecated function: CRM_Groupreg_DAO_GroupregEvent needs to be regenerated. Missing getEntityTitle method. in CRM_Core_Error::deprecatedWarning() (line 1053 of /var/www/d7civi47latest/sites/all/modules/civicrm/CRM/Core/Error.php).
User deprecated function: CRM_Groupreg_DAO_GroupregPriceField needs to be regenerated. Missing getEntityTitle method. in CRM_Core_Error::deprecatedWarning() (line 1053 of /var/www/d7civi47latest/sites/all/modules/civicrm/CRM/Core/Error.php).
User deprecated function: CRM_Groupreg_DAO_GroupregPriceField needs to be regenerated. Missing getEntityTitle method. in CRM_Core_Error::deprecatedWarning() (line 1053 of /var/www/d7civi47latest/sites/all/modules/civicrm/CRM/Core/Error.php).
```

The cause and solution should be essentially the same as in https://github.com/twomice/com.joineryhq.hideactivitytypes/issues/6

@lucky091588 Can you work up a PR?

**Comment by lucky091588** [comment]
@twomice Copy

**Comment by lucky091588** [comment]
@twomice Here is the PR: https://github.com/twomice/com.joineryhq.groupreg/pull/10

**Comment by twomice** [comment]
Thanks @lucky091588 

Fixed in #10
